src/main/java/org/minions/devfund/katerinaanzoleaga/moviedatabase/Actor.java

```diff
@@ -0,0 +1,85 @@
+package org.minions.devfund.katerinaanzoleaga.moviedatabase;
+
+import java.util.ArrayList;
+
+/**
+ * Actor class.
+ */
+public class Actor {
+    String name;
+    ArrayList<Movie> movies;
+
+    /**
+     * Actor class constructor, to be defined the way we are going to get the movies list
+     */
+    public Actor() {
+        this.name = "";
+        this.movies = new ArrayList<Movie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)